data/src/main/java/org/oppia/data/backends/gae/model/Topic.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Topic(
+
+  /*
+   * Ignore:
+   * is_moderator, is_admin, username, user_email, iframed, is_topic_manager, is_super_admin

```

Comment:
Let's just omit this comment. If we aren't including these, I don't think we need to include them.

Reply:
Removed this comment.